### PR TITLE
Add error message edit schedule when contact list is empty

### PR DESCRIPTION
**Pull request by tata32000**
This is added due to the actions failing silently as per the description in the course website.

image

image

It was also due to a bug in the code in the feature implemented in v1.5 (commit 385530e4367ed0743de0c2b233effd4d248784e8), at EditScheduleCommand.java line 114.

### CODE REVIEW

**Comment by tata32000** [comment]
Seems like this is not allowed because the error message is more specific.

**Comment by LimKaiWei** [comment]
yea this might be considered enhancements also

**Review by jewiteo — CHANGES_REQUESTED** [review]
LGTM but could you also add in the description of why this is a bug into the PR description.

File: src/main/java/seedu/address/logic/commands/EditScheduleCommand.java [hunk 1]
```diff
@@ -110,8 +114,11 @@ private static List<UUID> getUpdatedContactUids(Meeting meetingToEdit, List<UUID
             }
         }
 
-        // check if updated contact list is empty, if so, revert to previous contact list
-        updatedContactUids = updatedContactUids.isEmpty() ? previousContactUids : updatedContactUids;
```
[the remark this answers is not in the document]
Comment (tata32000) [reply src/main/java/seedu/address/logic/commands/EditScheduleCommand.java:114]: Yea, I think we should treat this as feature flaw and update DG, Thanks for the info!

**Pull request merged**